frontend/react/src/utils/AddCourse.jsx

```diff
@@ -3,7 +3,7 @@
 // Called by CreateCourse when a course form is submitted and sends the data to the backend
 export default function addCourseHandler(courseData) {
     fetch(
-        'http://localhost:5000/course',
+        '',
```

Comment:
Why was this removed? Either the urls in all functions should be updated to point to a valid server, or the urls should be left as localhost for now

Reply:
Just Updated it